lib/cocoapods/user_interface.rb

```diff
@@ -19,6 +19,7 @@ class << self
       attr_accessor :indentation_level
       attr_accessor :title_level
       attr_accessor :warnings
+      attr_accessor :io
```

Comment:
Regarding we only use it for output, seems reasonable, but I want to make sure it expects an IO object. I guess I can add type as documentation.